components/customs/arboStats.tsx

```diff
@@ -0,0 +1,15 @@
+"use client"
+
+import { useArboDataStatistics } from "@/hooks/useArboStatistics";
+
+function ArboStats() {
+    const arboDataStats = useArboDataStatistics();
```

Comment:
Big debate: should I instead be prefetching, dehydrating and rehydrating the data here? I technically do it one level up where this component is called, but do I need to do it here instead and convert this to a server component? Hmmm

Reply:
As far as I understand (and I am still navigating the waters of react query + SSR) what is happening here is the server calls the endpoint and gets the data. When loaded on the client, the client side cache is hydrated with the data that was fetched from the server (sent over as part of the defydratedState variable when site loads) and is hence loaded from cache

The only reason I wondered if I should do it here is the instance this component is accessed from another page we would have to hydrate it there seperately, but then again not sure. 

I think from this discussion I believe it best to dehydrate and rehydrate here to make the component truly reusable

Reply:
I am an idiot, we will never need this on the client side, I can just fetch on the server, populkate the value and send a static component to the client LOL